OasisDEX/oasis-borrow
# Add TBTC/WBTC Ajna pool

# [Add TBTC/WBTC Ajna pool](https://app.shortcut.com/oazo-apps/story/10439/add-tbtc-wbtc-to-ajna-curated-pools)

Ignore branch name, GHO token is not supported in this PR as there is no GHO/DAI pool available yet.

![image](https://github.com/OasisDEX/oasis-borrow/assets/16230404/e4d1b31f-fa2a-46c5-9007-6f7e12a13325)
  
## Changes 👷‍♀️

- Added support for TBTC coin in the app,
- added suport TBTC/WBTC Ajna pool,
- replaced WBTC in Product Finder with BTC group.
  
## How to test 🧪

- Run Product Finder PATCH,
- look for TBTC/WBTC products,
- check open UI.

Also, ignore the duplication, I had to add `primaryTokenGroup: 'BTC',` in a lot of places.
